src/components/client/notifications/NotificationList.tsx

```
@@ -59,14 +82,42 @@ export default function NotificationList() {
           {showIgnored ? "隱藏已忽略" : "顯示已忽略"}
         </button>
       </div>
-      {filteredNotifications.map((notification: any, index: number) => (
+      {filteredNotifications.map((notification: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)